dask/array/routines.py

```diff
@@ -1403,17 +1404,61 @@ def piecewise(x, condlist, funclist, *args, **kw):
     )
 
 
+def aligned_coarsen_chunks(chunks: List[int], multiple: int) -> List[int]:
+    """ Returns a new chunking aligned with the coarsening multiple"""
```

Comment:
Can I ask for a couple of simple docstring examples here.  I find that that is an easy way to quickly convey intent in situations like these.